js/components/MilkdropWindow/index.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import butterchurn from "butterchurn";
+import butterchurnPresets from "butterchurn-presets";
+import GenWindow from "../GenWindow";
+
+class MilkdropWindow extends React.Component {
+  constructor() {
+    super();
+  }
+  componentDidMount() {
+    const analyserNode = this.props.mediaPlayer._analyser;
+    const gainNode = this.props.mediaPlayer._gainNode;
+    this.visualizer = butterchurn.createVisualizer(
+      analyserNode.context,
+      this._canvasNode,
+      {
+        width: this._canvasNode.width,
+        height: this._canvasNode.height
```

Comment:
Do you have any way to tell Butterchurn that the canvas has been resized?

Reply:
Yes, setRendererSize(width, height), will update the renderer